src/app/(account)/settings/page.tsx

```diff
@@ -14,11 +14,21 @@ export const metadata: Metadata = {
 };
 
 export default async function SettingsPage() {
-    const user = await currentUser();
-    if (!user) return notFound();
+    const session = await auth();
+    var exists = false;
+    var membershipPayment:
+        | { paid: true; membershipExpiresAt: Date }
+        | { paid: false; membershipExpiresAt?: undefined } = {
+        paid: false,
+        membershipExpiresAt: undefined,
+    };
```

Comment:
Use `let`, not `var`. Additionally I wonder if it would be cleaner to define this as a separate type rather than inline like this.